example/distill/nlp/README.md

```diff
@@ -0,0 +1,50 @@
+# ERNIE distillation
+
+## Dataset
```

Comment:
description is required to make this task clear, not only the dataset. That is what this task is.

Reply:
Done.